python-template/{{cookiecutter.placeholder_repo_name}}/openedx.yaml

```diff
@@ -0,0 +1,17 @@
+# openedx.yaml
```

Comment:
[OEP-2](https://github.com/edx/open-edx-proposals/blob/master/oeps/oep-0002-bp-repo-metadata.rst) has been amended, most significantly in the `owner` key.

Consider updating this. It would require asking the user whether this repo will be owned by GitHub team or another repository, and then asking for that team/repo.